lib/rexml/parsers/baseparser.rb

```diff
@@ -766,6 +771,19 @@ def process_instruction
         [:processing_instruction, name, content]
       end
 
+      if StringScanner::Version < "3.1.1"
+        def get_quote
+          @source.match(/(['"])/, true)&.[](1)
+        end
+      else
+        def get_quote
```

Comment:
Can we use better name than this? `match_quote`? `scan_quote`?

Reply:
`scan_quote` looks good.
Thanks.